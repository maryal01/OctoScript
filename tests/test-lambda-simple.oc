lambda<int, int => int> iadd = fn (int i, int j) => {i + j};
print("%d, ", iadd(5, 15));

lambda<float, float => float> fadd = fn (float i, float j) => {i + j};
print("%f", fadd(41.9841, 1381.2438));
