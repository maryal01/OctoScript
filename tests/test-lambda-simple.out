20, 1423.227900